src/storage-client/src/types/sources.proto

```diff
@@ -200,11 +200,12 @@ message ProtoPostgresSourceConnection {
     mz_repr.global_id.ProtoGlobalId connection_id = 6;
     mz_storage_client.types.connections.ProtoPostgresConnection connection = 1;
     string publication = 2;
-    ProtoPostgresSourceDetails details = 4;
+    ProtoPostgresSourcePublicationDetails details = 4;
     repeated ProtoPostgresTableCast table_casts = 5;
+    repeated uint64 table_cast_pos = 7;
```

Comment:
Is this representing the selected tables? The same seems a bit hard to parse. It's the position of table casts?

Reply:
It's the position in the PG source to which the table cast correlates. This is called the "output index" elsewhere. If you have a suggestion for an improved name, glad to adopt it, or can be more explicit in the code about the field's intended semantics.